rocky/tools/forms/base.py

```diff
@@ -60,14 +60,21 @@ def format_value(self, value):
 
 
 class ObservedAtForm(BaseRockyForm):
-    observed_at = forms.DateField(
+    observed_at = forms.DateTimeField(
         label=_("Date"),
-        widget=DateInput(format="%Y-%m-%d"),
+        widget=DateTimeInput(format="%Y-%m-%d"),
         initial=lambda: datetime.datetime.now(tz=datetime.timezone.utc),
-        required=True,
+        required=False,
         help_text=OBSERVED_AT_HELP_TEXT,
     )
 
+    def clean_observed_at(self):
+        observed_at = self.cleaned_data["observed_at"]
+        now = datetime.datetime.now(tz=datetime.timezone.utc).date()
+        if observed_at > now:
+            raise forms.ValidationError(_("Your selected date is in the future. Please select a different date."))
```

Comment:
Given that observed_at is changed to a datetime, we should compare it to the current date and time and not just the current date.